includes/process-download.php

```diff
@@ -233,13 +233,25 @@ function edd_process_download() {
 
 				}
 
+				// Determine if the file we're attempting to use direct on is too large for the memory limit to use direct
+				$memory_limit = @ini_get( 'memory_limit' );
+				if ( $memory_limit ) {
+					$memory_limit_bytes = edd_determine_size_in_bytes( $memory_limit );
+					$file_size          = edd_determine_size_in_bytes( @filesize( ABSPATH . $file_path ) );
```

Comment:
Above we do `@filesize( $file_path )` so `ABSPATH` shouldn't be used here I don't think.